MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+recursive-include pls/static/.pls *
```

Comment:
Why not just `pls/static/`?

Reply:
Decided to rename it into some `dotpls`.

Reply:
rename to `dot_pls`